### PR TITLE
impl local bd for build screen

### PR DESCRIPTION
**Pull request by guvanch-amangeldiyev**

### CODE REVIEW

File: packages/data/lib/di/injector.dart [hunk 1]
```diff
@@ -98,4 +103,16 @@ Future<void> injectDataModule() async {
   sl.registerSingleton<PropertyApiMapper>(
     PropertyApiMapper(),
   );
+
+  //! database
+
+  sl.registerSingletonAsync<Database>(
+    () => LocalDBInit(nameDB: 'jenkis', version: 1).db,
```
Comment (ilya-safyan-idf) [inline packages/data/lib/di/injector.dart]: move nameDB and Version arguments to constants file.  This will make it easier to read..

**Review by ilya-safyan-idf — APPROVED** [review]

File: packages/data/lib/datasource/locally_data/repository/local_db_repository.dart [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  @override
+  Future<void> deleteLocalDBProperty(String nameJob) async {
+    try {
+      await db.delete(
+        'propertyOFJobs',
+        where: 'nameJob = ?',
+        whereArgs: [nameJob],
+      );
+    } catch (error) {
+      log(error.toString());
```
Comment (sergei-mikhailovskii-idf) [inline packages/data/lib/datasource/locally_data/repository/local_db_repository.dart:24]: error.printStackTrace()

File: packages/data/lib/datasource/locally_data/repository/local_db_repository.dart [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }
+  }
+
+  @override
+  Future<List<Map<String, dynamic>>> getLocalBuildResponse(
+      String property) async {
+    try {
+      return await db.rawQuery(LocalDBStructure.queryGetCacheJob(property));
+    } catch (e) {
+      log(e.toString());
```
Comment (sergei-mikhailovskii-idf) [inline packages/data/lib/datasource/locally_data/repository/local_db_repository.dart:55]: error.printStackTrace()

File: packages/data/lib/datasource/locally_data/repository/local_db_repository.dart [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      for (var element in listProperty) {
+        batch.insert(
+          "propertyOfJobs",
+          {'idDetailsJob': idRow, 'variant': element},
+        );
+      }
+
+      await batch.commit();
+    } catch (e) {
+      log(e.toString());
```
Comment (sergei-mikhailovskii-idf) [inline packages/data/lib/datasource/locally_data/repository/local_db_repository.dart:87]: error.printStackTrace()

File: packages/data/lib/datasource/locally_data/models/struct_of_db.dart [hunk 5]
```diff
@@ -0,0 +1,31 @@
+abstract class LocalDBStructure {
+  static String getPath(
+    String databasesPath,
+    String nameDB,
+  ) =>
+      '$databasesPath/$nameDB.db';
+
+  static String queryGetCacheJob(String job) {
```
Comment (sergei-mikhailovskii-idf) [inline packages/data/lib/datasource/locally_data/models/struct_of_db.dart:8]: remove static

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

**Pull request merged**